src/config/media/pipewire.md

```diff
@@ -0,0 +1,66 @@
+# PipeWire
+
+To use PipeWire, install the `pipewire` package.
+
+[pipewire(1)](https://man.voidlinux.org/pipewire.1) should be started as a user.
+Use the autostarting mechanism of your desktop environment or
+[startx](../graphical-session/xorg.md#startx).
+
+## PulseAudio replacement
+
+PipeWire can start a PulseAudio server. While it's possible to modify
+[pipewire.conf(5)](https://man.voidlinux.org/pipewire.conf.5) for auto-starting
+the PulseAudio server, it's recommended to start it from the same place where
+you start PipeWire by executing `pipewire-pulse`. This keeps PipeWire's
```

Comment:
"PipeWire's configuration file" sounds a bit weird to me, what about "the PipeWire configuration file"? Keep in mind that I am not a native speaker, so I might be wrong.

Reply:
Correct, I'll update this